### PR TITLE
usb: Cleanup for multiplied defines

### PR DESCRIPTION
**Pull request by eobalski**
Some of defines are present in several header files.
Those defines are the same with value but with different naming.

Common defines are brought to usb_common.h

Signed-off-by: Emil Obalski <emil.obalski@nordicsemi.no>

### CODE REVIEW

File: include/usb/usb_common.h [hunk 1]
```diff
@@ -65,10 +65,14 @@
 #define USB_INTERFACE_DESC		0x04
 #define USB_ENDPOINT_DESC		0x05
 #define USB_DEVICE_QUAL_DESC		0x06
+#define USB_OTHER_SPEED			0x07
+#define USB_INTERFACE_POWER		0x08
 #define USB_INTERFACE_ASSOC_DESC	0x0B
 #define USB_DEVICE_CAPABILITY_DESC	0x10
 #define USB_HID_DESC			0x21
 #define USB_HID_REPORT_DESC		0x22
+#define CS_INTERFACE			0x24
```
Comment (jfischer-phytec-iot) [inline include/usb/usb_common.h]: Please rename to USB_CS_INTERFACE_DESC and USB_CS_ENDPOINT_DESC

**Review by jfischer-no — CHANGES_REQUESTED** [review]

File: include/usb/usb_common.h (hunk 1 above)
Comment (emob-nordic) [reply include/usb/usb_common.h]: done

**Review by jfischer-no — APPROVED** [review]

File: subsys/usb/class/cdc_acm.c [hunk 2]
```diff
@@ -118,15 +118,15 @@ struct usb_cdc_acm_config {
 #define INITIALIZER_IF_HDR						\
```
Comment (finikorg) [inline subsys/usb/class/cdc_acm.c]: I think we also can move initializers to the common header.

**Review by finikorg — APPROVED** [review]

**Review by nashif — APPROVED** [review]

File: subsys/usb/class/cdc_acm.c (hunk 2 above)
Comment (emob-nordic) [reply subsys/usb/class/cdc_acm.c]: For some yes, but there exist a group of class specific ones which better stay here. We can consider this change later and decide they best solution.

**Review by carlescufi — APPROVED** [review]

**Pull request merged**